api/app.py

```diff
@@ -11,10 +11,13 @@
     ParseFromDuckDB,
 )
 
+from shareable import MongoDBConnection, PlanData
+
 from loguru import logger
 
 app = FastAPI()
-con = None
+duckConn = None
+mongoConn = MongoDBConnection()
```

Comment:
`duck_conn` and `mongo_conn`?

Reply:
changes the mogo_conn object. For duckConn, will make the change in https://github.com/voltrondata/substrait-fiddle/pull/32